.github/workflows/release_please.yml

```diff
@@ -54,3 +54,40 @@ jobs:
           skip_existing: true
         env:
           CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Get PR data
+        uses: actions/github-script@v7
+        if: startsWith(github.event.head_commit.message, 'chore(main): Release')
+        id: get_pr_data
+        with:
+          script: |
+            return (
+              await github.rest.repos.listPullRequestsAssociatedWithCommit({
+                commit_sha: context.sha,
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+              })
+            ).data[0];
+
+      - name: Update PR labels
+        uses: actions/github-script@v7
+        if: startsWith(github.event.head_commit.message, 'chore(main): Release')
+        with:
+          script: |
+            const prNumber = ${{ fromJson(steps.get_pr_data.outputs.result).number }};
```

Comment:
Hopefully this works, otherwise we need to pass it as env input https://github.com/actions/github-script?tab=readme-ov-file#use-env-as-input

Reply:
My IDE doesn't like it, but it could be wrong.

Reply:
It should work https://github.com/actions/github-script/issues/56#issuecomment-639002362

Reply:
Also, another IDE complaint is

`if: startsWith(github.event.head_commit.message, 'chore(main): Release')`

Should be

```
if: |
  startsWith(github.event.head_commit.message, 'chore(main): Release')
```

Reply:
The GitHub UI tells you if the file has a syntax error when you open it https://github.com/cloudquery/helm-charts/blob/chore/tidy-up-pr-tags/.github/workflows/release_please.yml

Reply:
OK I think we need multiline strings for the `if` conditions